app/views/posts/show.html.erb

```diff
@@ -1,18 +1,29 @@
 <section class="post">
   <div class="post__detail">
     <div class="post__detail--body">
-      <h3>Post #<%= @post.id%> by <%= @post.user.name %></h3>
+      <h3>Post #<%= @post.id %> by <%= @user.name %></h3>
```

Comment:
- [ ] Same here, this part of the code is commented out, using `#` means comment in Ruby, hence it has introduced a bug in our code as the other one